### PR TITLE
PR for Point to Point HVDC model

### PR DESCRIPTION
**Pull request by Carlos-Alegre**
Pull request for the ID-190 Point to Point EMT HVDC model

### CODE REVIEW

File: pages/models/HVDC/EMTVSC/index.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+The model presented is valid to perform EMT simulations of an HVDC link, (a point-to-point HVDC transmission line). It is valid to perform studies such as lightning events, control dynamics, short-circuit studies and other high-frequency phenomena. The following assumptions are made:
+
+* There are two converters at both ends of the line. The model showcases a Grid Forming - Grid Following configuration (although there could be different configurations).
+* The DC line is represented by a resistance and one capacitor at each end that are useful for the transient stability of the line.
+* Each of the converters follows the same assumptions as in the individual models, available here: [EMT Grid Following](../../generations/Sources/VSC/EMTGridFollowingVSC/) and [EMT Grid Forming](../../generations/Sources/VSC/EMTGridFormingVSC/).
+
+As all the EMTs model, it is not suitable for low-frequency studies, such as power flow or dynamic stability studies, as its computational cost is high, and the time step cannot be reduced too much without loosing convergence. It is also not suitable for harmonic studies, as the PLLs only include the fundamental frequency.
```
Comment (GJCRESYM) [inline pages/models/HVDC/EMTVSC/index.md:24]: Typo: "losing" instead of loosing

File: pages/models/HVDC/EMTVSC/index.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+<div align = 'center'>
+Figure 2: Active power control diagram of the HVDC model
+</div>
+<br>
+
+The measured power exchanged with the AC grid $$P$$, with positive values for power injection, matches the power drawn from the AC grid, in this case it equals the injection demanded by the setpoint of the DC voltage control $$P_{DC}$$ plus the injection of the DC side. This DC power will be positive if there is generation in the Grid Forming side, while it would be negative if there is a load.
+
+## Open source implementations
+
+This model has been successfully implemented in :
```
Comment (GJCRESYM) [inline pages/models/HVDC/EMTVSC/index.md:87]: Table is not formatted properly

**Review by GJCRESYM — CHANGES_REQUESTED** [review]

File: pages/models/HVDC/EMTVSC/index.md (hunk 2 above)
Comment (Carlos-Alegre) [reply pages/models/HVDC/EMTVSC/index.md:103]: Corrected

File: pages/models/HVDC/EMTVSC/index.md (hunk 1 above)
Comment (Carlos-Alegre) [reply pages/models/HVDC/EMTVSC/index.md]: Corrected

**Review by GJCRESYM — APPROVED** [review]

**Pull request merged**